op-enable.go

```diff
@@ -380,3 +357,22 @@ func getArgs(s DockerHandlerSettings, dd driver.DistroDriver) string {
 
 	return strings.Join(args, " ")
 }
+
+
+func getDockerUrls(env string) (string, string) {
```

Comment:
can you return `struct{dockerUrl, composeUrl string}` from this method instead of two strings.

or better create a struct called `type dockerInstallEnv struct{dockerUrl, composeUrl string}`

and return `(dockerInstallEnv, error`). From a good coding practices perspective:

- You _should_ not have logic about default environment here
- This method should return error when env is not one of the valid values
- Please populate `env` with the default value before passing it down here.